_assets/js/netlify/topics/makeBodyContent.js

```diff
@@ -0,0 +1,14 @@
+const md = window.markdownit();
+md.options['html'] = true;
+md.options['linkify'] = true;
+const parser = new DOMParser();
+// This function takes our body content as input, turns the markdown into HTML,
+// then loops over the html elements
+const makeBodyContent = (bodyContent) => {
+  let html = parser.parseFromString(md.render(bodyContent), 'text/html');
```

Comment:
nit: const vs let, here and below

Reply:
Thanks! I default to let which is prolly backwards.